### ISSUE TITLE
Setting up Github repository with WP 11/25/19

### ISSUE THREAD

**Issue by rpmoses** [rpmoses/capstoneTeam6#6]
Professor Schofield sen an emial that explained how to set up git with wp. Once wp was set up I attempted to link the git repo with the wp in Ubuntu. Had issues when I was trying to clone my repo. The directions I used was using a Mac machine, so directions where hard to follow. I  continued to google other ways, but still got stuck on the command line directions. @matthewFobus can you check this out. 

Here is the link to the article I used: https://45royale.com/blog/wordpress-github/

**Comment by Mforbus1** [comment]
> sudo git clone [insert link to repository] 
in the WordPress theme folder

In WordPress, select this theme.

> sudo git add . 
inside of the cloned repository to add the files to the commit process

> sudo git commit -m "insert commit message"
commits changes with the message inserted

> sudo git push
pushes changes to the repository after inputting your github information
